checks/microbenchmarks/gpu/memory_bandwidth/memory_bandwidth.py

```diff
@@ -3,14 +3,11 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
-import os
-import sys
-
 import reframe as rfm
-from hpctestlib.microbenchmarks.gpu.memory_bandwidth import *
 
-import microbenchmarks.gpu.hooks as hooks
+from hpctestlib.microbenchmarks.gpu.memory_bandwidth import *
 
+hooks = rfm.utility.import_module("..hooks")
```

Comment:
I prefer this one to be
hooks = rfm.utility.import_module("....microbenchmarks.gpu.hooks")